ui.py

```diff
@@ -1388,7 +1477,15 @@ def create_shader_nodes(material_name, mesh_materials_path, textures_path):
     normal_map_node = nodes.new(type="ShaderNodeNormalMap")
     set_node_position(normal_map_node, 12, 10)
 
-    principled_node.inputs[27].default_value = 100
+    # Removing this fixed an issue loading the materials from the dds files. Idk why this was here.
```

Comment:
I'm not sure what this was supposed to do. If we need emissions (which is what I think this was), then I've provided an example of how we could do that.